### ISSUE TITLE
Support for embedded TeX

### ISSUE THREAD

**Issue by pinkcreeper100** [artempyanykh/marksman#143]
e.g $\frac{1}{2}$

**Comment by artempyanykh** [comment]
Hi @pinkcreeper100! What would you like the support to look like? Any examples of assists?

**Comment by pinkcreeper100** [comment]
I wouldn't mind if it was just some sort of highlighting when it is surrounded by $

**Comment by pinkcreeper100** [comment]
Although other than that, I'm not entirely sure

**Comment by artempyanykh** [comment]
Right! I think the highlighting can be done with LSP semantic tokens. However, semantic tokens (last I checked) were not yet supported in neovim and Emacs/Eglot; not sure about helix.

**Comment by keynmol** [comment]
> were not yet supported in neovim

they are in nightlies (I'm using them in my LSPs), previous versions would probably just ignore the capability even if you send it back.
